background/background.js

```diff
@@ -581,15 +549,13 @@ if (typeof chrome.commands !== "undefined") {
     })
 }
 
-twpConfig.onReady(function () {
+twpConfig.onReady(async () => {
```

Comment:
Any special reasons to use async here?

Reply:
This function doesn't have to work synchronous, so asynchronous can give better performance I think